components/_icons.html

```diff
@@ -1,36 +1,84 @@
-<svg style="position: absolute; width: 0; height: 0; overflow: hidden;" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+<svg aria-hidden="true" style="position: absolute; width: 0; height: 0; overflow: hidden;" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
         <defs>
-        <symbol id="i-oval" viewBox="0 0 32 32">
-        <title>oval</title>
-        <path d="M29 16c0 7.18-5.82 13-13 13s-13-5.82-13-13c0-7.18 5.82-13 13-13s13 5.82 13 13z"></path>
-        </symbol>
-        <symbol id="i-cross" viewBox="0 0 32 32">
```

Comment:
I wasn't able to find usages of i-oval, i-cross, or i-polygon anywhere in the pattern lib, but they are in the webapp now that I look, under reserved units designs that aren't really functional right now anyways. 🤔 . I'll see if I can add them back

Reply:
@akegan should I wait to review til these are added back?

Reply:
@james-wills-sf yep. It looks like the files never actually existed, but I just went back to copying just the new icons from demo.html to _icons.html so we didn't lose those icons. I tried adding the oval, cross, and polygon icons to the selection.json file manually but that didn't work either